src/main/java/apoc/export/json/JsonFormatSerializer.java

```diff
@@ -68,10 +71,13 @@ public void serializeProperty(JsonGenerator jsonGenerator, String key, Object va
         private void writeNodeDetails(JsonGenerator jsonGenerator, Node node, boolean withNodeProperties) throws IOException {
             jsonGenerator.writeStringField("id", String.valueOf(node.getId()));
             Iterable<Label> labels = node.getLabels();
+
             if (labels.iterator().hasNext()) {
```

Comment:
```suggestion
            if (node.getLabels().iterator().hasNext()) {
```
e rimuovere la variabile sopra